src/lib/components/composites/user-avatar/UserAvatar.svelte

```diff
@@ -0,0 +1,64 @@
+<script lang="ts">
+    import * as Avatar from "$lib/components/primitives/avatar/index.js";
+    import { ReviewDecision, type User } from "$lib/types";
+    import Check from "lucide-svelte/icons/check";
+    import X from "lucide-svelte/icons/x";
+
+    interface Props {
+        user: User;
+        reviewDecision?: ReviewDecision;
+    }
+
+    const { user, reviewDecision }: Props = $props();
+
+    const getInitial = (text: string) => (text.length > 0 ? text[0].toUpperCase() : "");
+    const userInitials = `${getInitial(user.firstName)}${getInitial(user.lastName)}`;
+</script>
+
+<!--
+@component
+Avatar displaying the initials of the user or a profile image.
+
+This avatar component displays the set profile image or, if
+no image was set, the initials of the user. For instance, the
+name of the user is 'Max Mustermann', this avatar will display
+the initials 'MM'.
+
+Optionally, a reviewDecision can be added, which visualizes
+a decision of the associated user, which can be used for example
+in the PaperEntry component.
+
+Usage:
+```svelte
+    <UserAvatar user={{
+        firstName: "Max",
+        lastName: "Mustermann",
+    }} reviewDecision: ReviewDecision.Accept />
+```
+-->
+<div class="relative">
+    <Avatar.Root>
+        <Avatar.Fallback class="group-hover/paper-list-entry:bg-white"
+            >{userInitials}</Avatar.Fallback
+        >
```

Comment:
Minor, but this weirds me out 😅 
```suggestion
        <Avatar.Fallback class="group-hover/paper-list-entry:bg-white">
            {userInitials}
        </Avatar.Fallback>
```

Reply:
Me too, I changed it.